resources/carrier/en/213.txt

```diff
@@ -12,9 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-21354|Nedjma
-21355|Nedjma
-21356|Nedjma
+21354|Ooredoo
```

Comment:
A comment about the rebranding would be good here.
E.g. "Ooredoo used to be known as Nedjma." 

Please add it before this line and add line break after that.